ansible/roles/st2_dev/tasks/mongo.yml

```diff
@@ -2,16 +2,14 @@
 - name: Add mongo ppa key
   become: yes
   apt_key:
-    keyserver: hkp://keyserver.ubuntu.com:80
-    id: EA312927
+    url: https://www.mongodb.org/static/pgp/server-3.4.asc
     state: present
 - name: Add mongo sources list
   become: yes
-  lineinfile: >
```

Comment:
I used copy instead of ``lineinfile`` since liniinfile is not great fit for this problem - it will append file if it doesn't already exist, but it won't delete any existing / old entries so ``copy`` is a better fit for here.